### PR TITLE
add Sharepod

### PR DESCRIPTION
**Pull request by markthetech**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [x] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

### CODE REVIEW

File: Casks/sharepod.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'sharepod' do
+  version :latest
+  sha256 :no_check
+
+  # cdn.macroplant.com/release/Sharepod-4.3.2.0.dmg was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/sharepod.rb]: Too specific. Make it just `cdn.macroplant.com`.

File: Casks/sharepod.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  url 'http://cdn.macroplant.com/release/Sharepod-4.3.2.0.dmg'
```
Comment (vitorgalvao) [inline Casks/sharepod.rb]: `url "http://cdn.macroplant.com/release/Sharepod-#{version}.dmg"`

File: Casks/sharepod.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask 'sharepod' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/sharepod.rb]: `version '4.3.2.0'`

File: Casks/sharepod.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/sharepod.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by markthetech** [comment]
all fixed

File: Casks/sharepod.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+cask 'sharepod' do
+  version '4.3.2.0'
+  sha256 '1e6799be8764fb6f57ff87fef79de86d0391d63d349f68999f02729a72529211'
+
+  # cdn.macroplant.com was verified as official when first introduced to the cask
+  url 'http://cdn.macroplant.com/release/Sharepod-#{version}.dmg'
```
Comment (vitorgalvao) [inline Casks/sharepod.rb]: These need to be double quotes.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Pull request merged**